services/clusters-keeper/src/simcore_service_clusters_keeper/constants.py

```diff
@@ -0,0 +1,15 @@
+from typing import Final
+
+from aws_library.ec2._models import AWSTagKey, AWSTagValue
+from pydantic import parse_obj_as
+
+DOCKER_STACK_DEPLOY_COMMAND_NAME: Final[str] = "private cluster docker deploy"
+DOCKER_STACK_DEPLOY_COMMAND_EC2_TAG_KEY: Final[AWSTagKey] = parse_obj_as(
+    AWSTagKey, "io.simcore.clusters-keeper.private_cluster_docker_deploy"
+)
+
+USER_ID_TAG_KEY: Final[AWSTagKey] = parse_obj_as(AWSTagKey, "user_id")
```

Comment:
Q: shoulnd all AWS tags be prefixed by a reverse url like `io.simcore` etc ... to avoid conflicts?

Reply:
in theory yes, but I now probably need a migration scheme or a backwards compatibility scheme